opensrp-app/src/main/java/org/smartregister/broadcastreceivers/OpenSRPClientBroadCastReceiver.java

```diff
@@ -62,6 +67,15 @@ public void onReceive(Context context, Intent intent) {
 
     }
 
+    /**
+     * This method is invoked everytime that the device is shut down
+     *
+     * @param time the timestamp
+     */
+    protected void onDeviceShutdown(long time) {
```

Comment:
Why is this `protected` Why not use private?

Reply:
This is visible for testing. I can add the annotation `@VisibleForTesting` so that it is more clear

Reply:
Yes, please add the annotation.